build.mjs

```diff
@@ -75,29 +76,160 @@ const ES_BUILD_OPTIONS = {
+//
+//    Case) @aws-sdk/client-sts
+//    [Service endpoints Section](https://docs.aws.amazon.com/general/latest/gr/sts.html#sts_region)
+//
+//    > | Region Name    | Region    | Endpoint                    | Protocol |
+//    > | -------------- | --------- | --------------------------- | -------- |
+//    > | US East (Ohio) | us-east-2 | sts.us-east-2.amazonaws.com | HTTPS    |
+//                                     ^^^ <- String before "region"
+//
+//    If multiple endpoints are required, such as @aws-sdk/client-sso, register multiple endpoints in the array.
+//
+//   4. fullSdkOnly
+//
+//     If you want to bundle only 'full-sdk', specify `true`.
+//     Specify `false` if you want to bundle for both 'std-sdk' and 'full-sdk'.
+//
+//     The combination of SDK_BUNDLE_MODE and fullSdkOnly automatically determines whether the bundle is eligible or not.
+//     Note that if SDK_BUNDLE_MODE is 'NONE', the above values are completely ignored and any SDKs are excluded from the bundle.
+//
 const SDK_DATA = {
-  "client-dynamodb": ["DynamoDB", "dynamodb"],
-  "lib-dynamodb": ["DynamoDBDocument", "dynamodb"],
-  "client-kms": ["KMS", "kms"],
-  "client-lambda": ["Lambda", "lambda"],
-  "client-s3": ["S3", "s3"],
-  "lib-storage": ["Upload", "s3"],
-  "client-secrets-manager": ["SecretsManager", "secretsmanager"],
-  "client-ses": ["SES", "email"],
-  "client-sns": ["SNS", "sns"],
-  "client-sqs": ["SQS", "sqs"],
-  "client-sts": ["STS", "sts"],
-  "client-ssm": ["SSM", "ssm"],
-  "client-cloudwatch-logs": ["CloudWatchLogs", "logs"],
-  "client-cloudwatch-events": ["CloudWatchEvents", "events"],
-  "client-eventbridge": ["EventBridge", "events"],
-  "client-sfn": ["SFN", "states"],
-  "client-xray": ["XRay", "xray"],
-  "client-cognito-identity": ["CognitoIdentity", "cognito-idp"],
+  // Analytics
+  "client-athena": ["Athena", ["athena"], true],
+  "client-firehose": ["Firehose", ["firehose"], true],
+  "client-glue": ["Glue", ["glue"], true],
+  "client-kinesis": ["Kinesis", ["kinesis"], true],
+  "client-opensearch": ["OpenSearch", ["es"], true],
+  "client-opensearchserverless": ["OpenSearchServerless", ["aoss"], true],
+  // ApplicationIntegration
+  "client-eventbridge": ["EventBridge", ["events"], false],
+  "client-scheduler": ["Scheduler", ["scheduler"], true],
+  "client-sfn": ["SFN", ["states", "sync-states"], false],
+  "client-sns": ["SNS", ["sns"], false],
+  "client-sqs": ["SQS", ["sqs"], false],
+  // Blockchain
+  ...{},
+  // BusinessApplications
+  "client-ses": ["SES", ["email"], false],
+  "client-sesv2": ["SESv2", ["email"], true],
+  // CloudFinancialManagement
+  ...{},
+  // ComputeServices
+  "client-auto-scaling": ["AutoScaling", ["autoscaling"], true],
+  "client-batch": ["Batch", ["batch"], true],
+  "client-ec2": ["EC2", ["ec2"], true],
+  "client-lambda": ["Lambda", ["lambda"], false],
+  // CustomerEnablement
+  ...{},
+  // Containers
+  "client-ecr": ["ECR", ["ecr", "api.ecr"], true],
```

Comment:
Why is there two arrays here? `["ecr", "api.ecr"]`

Reply:
The definition is based on the description in the following URL.

https://docs.aws.amazon.com/general/latest/gr/ecr.html

> The ecr and api.ecr endpoints are used for calls to the Amazon ECR API. API actions such as DescribeImages and CreateRepository go to this endpoint. While the two endpoints function the same, the api.ecr endpoint is recommended and the default when using the AWS CLI or AWS SDKs. When connecting to Amazon ECR through an AWS PrivateLink VPC endpoint, you must use the api.ecr endpoint to make API calls.

Reply:
This looks like it could be unified with `api.ecr`.

There seems to be something else.
https://docs.aws.amazon.com/general/latest/gr/step-functions.html

> client-sfn": ["SFN", ["states", "sync-states"], false],